packages/devtools/package.json

```diff
@@ -39,9 +39,9 @@
     "prepublishOnly": "package-check"
   },
   "dependencies": {
-    "@ceramicnetwork/common": "^2.25.0",
-    "@ceramicnetwork/stream-model": "^1.7.0",
-    "@ceramicnetwork/streamid": "^2.14.0",
+    "@ceramicnetwork/common": "2.30.0-rc.0",
```

Comment:
Why is `common` saved as exact version and not the others please? That could create issues with type mismatches.

Reply:
Since it's an RC, was doing exact versions as a way to force them to be changed once a release is out. They should all be exact version. Will make sure I've done that.